socorro/processor/signature_utilities.py

```diff
@@ -732,8 +723,16 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
                 'Error parsing AsyncShutdownTimeout: {}'.format(e)
             )
 
-        processed_crash['signature'] = ' | '.join(parts)
-        return result
+        new_sig = ' | '.join(parts)
+        processor_meta['processor_notes'].append(
+            'Signature replaced with a Shutdown Timeout error, '
```

Comment:
"error"? I struggle to understand this sentence. Should it be "Signature replaced by a Shutdown Timeout signature"?